tests/Earthfile

```diff
@@ -147,6 +147,7 @@ ga-no-qemu-group5:
     BUILD +test-aws-flag-envs
     BUILD +test-aws-flag-configs
     BUILD +test-aws-flag-none
+    BUILD +test-aws-oidc
```

Comment:
I would put this under the last `group12` instead _if it didn't require any special auth_ (this is to keep the macstadium instances from becoming overloaded)

However, since it looks like it does require special credentials, it should be moved to the `tests-that-require-earthly-technologies-account-access` target

Reply:
i just scrolled down and saw a different test triggered via `require-earthly-technologies-account-access`, so then this might belong under `group12` :laughing: